### PR TITLE
Created waiting for other players page

### PR DESCRIPTION
**Pull request by geneh61**
Created skeleton of waiting for other players page w/o show answer logic implementation

### CODE REVIEW

**Review by goyeet — APPROVED** [review]

**Comment by yesh0907** [comment]
is this PR still active? if so, can the branch you are working on to the latest changes on main so we can see what commits are new?

**Comment by geneh61** [comment]
> if so, can the branch you are working on to the latest changes on main so we can see what commits are new?

can you clarify this?

**Comment by justinqbui** [comment]
> > if so, can the branch you are working on to the latest changes on main so we can see what commits are new?
> 
> can you clarify this?

You need to rebase your current branches on main. Your current commit is showing duplicate commits that have been merged in main from previous PRs. This makes it hard to evaluate the diff, since some of the changes already exist in main

File: src/routes/player/WaitingForOtherPlayers.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        </div>
+      </div>
+    </>
+  )
+}
+
+function WaitingForOtherPlayers() {
+
+  enableReactTracking({ auto: true });
+  persistObservable(state$, {
```
Comment (yesh0907) [inline src/routes/player/WaitingForOtherPlayers.tsx]: why do we need use this?

**Review by yesh0907 — APPROVED** [review]
code looks good but just checking why we need to use `ObservablePersistLocalStorage` plugin

File: src/routes/player/WaitingForOtherPlayers.tsx (hunk 1 above)
Comment (geneh61) [reply src/routes/player/WaitingForOtherPlayers.tsx]: I don't believe it is necessary, removed

**Pull request merged**